test/test_um2netcdf.py

```diff
@@ -395,7 +403,12 @@ def __init__(self, item_code, var_name=None, attributes=None, units=None):
         self.units = None or units
         self.standard_name = None
         self.long_name = None
-        self.coord = {}
+
+    def coord(self):
+        raise NotImplementedError(
+            "coord() method not implemented on DummyCube"
+        )
+
```

Comment:
The linter complained about the extra line break here.

I've added #97 as a reminder for sorting out code auto formatting.

Reply:
Added in!